VCFv4.3.tex

```diff
@@ -1593,6 +1594,7 @@ \subsubsection{Type encoding}
 \vspace{0.3cm}
 \begin{tabular}{|l | l | l|} \hline
 Lowest 4 bits & Hexadecimal encoding & Corresponding atomic type \\ \hline
+0 & 0x00 & A missing value without an explicit type provided \\ \hline
```

Comment:
Use `MISSING` to show that this is talking about the particular VCF/BCF meaning of _missing_.